pkg/data/domain/cluster/error.go

```diff
@@ -0,0 +1,41 @@
+package cluster
+
+import (
+	"github.com/giantswarm/microerror"
+)
+
+var invalidConfigError = &microerror.Error{
+	Kind: "invalidConfigError",
+}
+
+// IsInvalidConfig asserts invalidConfigError.
+func IsInvalidConfig(err error) bool {
+	return microerror.Cause(err) == invalidConfigError
+}
+
+var noResourcesError = &microerror.Error{
```

Comment:
Listing clusters (without an ID) is handled with a custom error because in certain situations we have to handle it differently:
* when using default output type (no `--output`), the process should return the code 0 (no error)
* when using any other output type (e. g. `--output json`), the process should return a non 0 code (error)